src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,178 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size = 0;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+    }
```

Comment:
No need to set head and tail null. They are null by default.
No need to create this constructor. It will be created by default because it is the only constructor in this class.
```suggestion
```